pkgs/by-name/li/libei/package.nix

```diff
@@ -67,6 +67,6 @@ stdenv.mkDerivation rec {
     homepage = "https://gitlab.freedesktop.org/libinput/libei";
     license = licenses.mit;
     maintainers = [ maintainers.pedrohlc ];
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
This shouldn't be here, because `libei` is disabled in the Input Leap package when Darwin is detected. Furthermore, `libei` does not support Darwin.

Reply:
Oh whoops, that was left over from experimenting